file-formats/ilmb/README.md

```diff
@@ -0,0 +1,5 @@
+# ILMB File Format
+
+File | Cardinality | Definition | Schema | Example
+:--- | :---  | :--- | :--- | :---
+`<name>.ilmb.json` | 1 | [`zif_aff_ilmb_v1.intf.abap`](./type/zif_aff_ilmb_v1.intf.abap) | [`ilmb-v1.json`](./ilmb-v1.json) | [`z_aff_example_ilmb.ilmb.json`](./examples/z_aff_example_ilmb.ilmb.json)
```

Comment:
You have either to add the example or remove the link. This fixes the broken link build error.